nengo/params.py

```diff
@@ -304,9 +304,7 @@ def __init__(self, name, default=Unconfigurable, length=None, low=0,
         self.low = low
 
     def __set__(self, instance, value):
-        try:
-            value = tuple(int(v) for v in value)
-        except TypeError:
+        if not all(is_integer(v) for v in value):
```

Comment:
So the reason I didn't do this, is that I often do something like `np.floor(a / b)` to get some of my shape indices, and it's just convenient to not have to explicitly cast to an int. That said, maybe it's better to have to be explicit. But would it be worth checking if a floating point number is actually exactly an integer, and in that case just doing the cast ourselves?